index.html

```diff
@@ -45,6 +45,34 @@ <h2 class="font2">I'm a Software Developer</h2>
                 <li><a href="https://medium.com/"><img src="assets/mobile/medium.svg" alt="medium icon that will link to Medium page"></a></li>
             </ul>
         </section>
+        <section class="modal modal-close">
+            <div class="modal-container d-flex">
+                <div class="modal-title d-flex">
+                        <div class="name-container d-flex">
+                            <h3 class="name title">Multi Post Stories</h3>
+                            <img src="assets/mobile/close-Icon.svg" alt="close button" class="close-button">
+                        </div>
+                        <ul class="tech-container skill-lists border-1">
+                            <li>HTML</li>
+                            <li>Bootstrap</li>
+                            <li>Ruby on Rails</li>
+                        </ul>
+                </div>
+                <div class="modal-body d-flex">
+                    <img class="modal-image"  src="assets/mobile/ImgPlaceholder-desktop.jpg" alt="image placeholder desktop">
+                    <div class="project-description-container">
+                        <div class="project-description d-flex font-body-reg">
+                            <p class="prj-description">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.</p>
+        
```

Comment:
- [ ] All these lines of code are supposed to be generated dynamically using JavaScript according to the requirement. Kindly fix it. 👍